utils/format

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+find examples -name "*.h"  -exec clang-format -i {} \;
+find examples -name "*.cc"  -exec clang-format -i {} \;
```

Comment:
SPIRV-Tools uses .cpp and .hpp files.

Reply:
I'd probably do a double loop

 for dir in source test tools include; do
  . ..
 ...

Reply:
or do something like this.

alias cgrep='find . -type f \( -name "*.c" -o -name "*.cc" -o -name "*.h" -o -name "*.hpp" -o -name "*.cpp" \) -print0 | xargs -0 grep'

Reply:
done